pontoon/messaging/templates/messaging/emails/monthly_activity_summary.html

```diff
@@ -106,11 +106,11 @@ <h2 class="team-activity">Team activity for {{ locale.name }} ({{ locale.code }}
 <p class="item {{ cls }}">
     <span class="bullet {{ url_parameter }}"></span>
     {{ label }}:
-    {% if url_parameter %}
+    {% if url_parameter and (locale.month_stats[attribute] or 0) > 0 %}
```

Comment:
It should be sufficient to do this:
```suggestion
    {% if url_parameter and locale.month_stats[attribute] %}
```

Which is also what @allie5 suggested in https://github.com/mozilla/pontoon/discussions/3568.